experimental/traffic-portal/src/app/core/core.module.ts

```diff
@@ -116,6 +116,7 @@ export const ROUTES: Routes = [
 	{ component: ProfileDetailComponent, path: "profiles/:id"},
 	{ component: ProfileTableComponent, path: "profiles"},
 	{ component: TopologyDetailsComponent, path: "topologies/:name"},
+	{ component: TopologyDetailsComponent, path: "topologies-new"},
```

Comment:
we have `new-role` and `new-capability`. We don't have rules for how to write the routes, and technically anything unique will work, but I'd suggest sticking with the phonetic kebab-case statement of purpose, so people know at a glance what a link is gonna take them to.

Reply:
Changed to `new-topology` in b175ed8689